cypress.config.js

```diff
@@ -1,9 +1,26 @@
 const { defineConfig } = require('cypress');
+const faker = require('faker');
 
 module.exports = defineConfig({
   e2e: {
+    baseUrl: 'https://demoqa.com',
+    viewportHeight: 1080,
+    viewportWidth: 1920,
     setupNodeEvents(on, config) {
-      // implement node event listeners here
+      on('task', {
+        generateUser() {
+          const randomIndex = Math.floor(Math.random() * 2);
+          const departments = ['Insurance', 'Compliance', 'Legal'];
+          return {
+            firstName: faker.name.firstName(),
```

Comment:
The user data generated is not being validated. It's recommended to add validation to ensure data integrity.